datanode/api/trading_data_v2.go

```diff
@@ -1364,7 +1364,7 @@ func (t *TradingDataServiceV2) ListParties(ctx context.Context, req *v2.ListPart
 	}, nil
 }
 
-// ListMarginLevels lists MarginLevels using a cursor based pagination model.
+// ListMarginLevels lists MarginLevels.
```

Comment:
```suggestion
// Lists margin levels.
```
for a given market, for all markets?